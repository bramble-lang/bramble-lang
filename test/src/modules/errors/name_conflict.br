
fn my_main() {
    let ms: MyStruct := my_mod::g();
    return;
}

struct MyStruct {
    b: i64,
}

fn f() -> MyStruct {
    return MyStruct{b: 6};
}

mod my_mod {
    struct MyStruct {
        a: i64,
    }

    fn g() -> MyStruct {
        return MyStruct{
            a: 5,
        };
    }
}