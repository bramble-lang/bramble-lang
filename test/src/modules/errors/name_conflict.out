Error: Semantic: L3: Bind expected root::MyStruct but got root::my_mod::MyStruct
